src/ledger.rs

```diff
@@ -399,15 +399,9 @@ impl LedgerWriter {
         Ok(())
     }
 
-    pub fn flush(&mut self) -> io::Result<()> {
```

Comment:
glad this is gone, but I want to preserve the "write_entry()" or "write_entries()" implies flush